src/VisualStudio/CSharp/Impl/Options/Formatting/StyleViewModel.cs

```diff
@@ -1452,6 +1452,25 @@ private void AddParenthesesOptions(OptionSet optionSet)
                 defaultAddForClarity: false);
         }
 
+        private void AddBracesOptions(OptionSet optionSet, string bracesPreferenceGroupTitle)
+        {
+            var bracesPreferences = new List<CodeStylePreference>
+            {
+                new CodeStylePreference(ServicesVSResources.Yes, isChecked: false),
+                new CodeStylePreference(ServicesVSResources.No, isChecked: false),
+                new CodeStylePreference(CSharpVSResources.When_on_multiple_lines, isChecked: false),
+            };
+
+            var enumValues = new[] { PreferBracesPreference.Always, PreferBracesPreference.None, PreferBracesPreference.WhenMultiline };
+
+            CodeStyleItems.Add(new EnumCodeStyleOptionViewModel<PreferBracesPreference>(
+                CSharpCodeStyleOptions.PreferBraces,
+                ServicesVSResources.Prefer_braces,
+                enumValues,
+                new[] { s_preferBraces, s_preferBraces, s_preferBraces },
```

Comment:
same preview for all values?

Reply:
[Currently yes](https://github.com/dotnet/roslyn/commit/a2af13b357f41918b76564e27fd2ccb38a1dcf72#diff-00b96c088d3f8c1ad44994240cc400e0) 😆